Worksheet B/Pseudocode.MD

```diff
@@ -0,0 +1,38 @@
+```
+
+
+Word Bank: Repulsive, Bubble, Men, Roasted, Sophisticated,
+Sturdy, Pretend, Nod, Group, Rat, Open, Dramatic, Etc…
+
+Set χ = 0
+
+Print 7 Random Words from Word Bank
+
+Assign 1 word as TruePassword
+
+Print ‘Enter Password’
+
+Read password
+
+If password =TruePassword
+
+Print ‘Correct!’
+
+Allow User Access
+
+Else if password = False
+
+Read TruePassword
+
+If password has
+a matching letter to TruePassword add
++1 to χ
```

Comment:
Need to make it clear that this is looping over the words, checking each letter in turn